### ISSUE TITLE
FNX3-17137 ⁃ [Bug] Not all the text from an error page is read while Talkback is on

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#3586]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Launch Fenix.
1. Home screen.
1. Add a wrong website **`ww.cnn.com`.**

### Expected behavior
The error message & "try again" button are read.

### Actual behavior
The word `Sections` is repeated for like 10+ times and the last two points 

* **`Check the address for typing errors such as ww.example.com instead of www.example.com.`**
* **`If you are unable to load any pages, check your devices' data or Wi-FI connection.`**
cannot be hear while TalkBack is on.
* Android device: **`Samsung Galaxy Note 9(Android 8.1.0).`**
* Fenix version: **`6/19 1.0.0-rc.2`**

**Comment by abodea** [comment]
Verified as fixed on the latest **Nightly** build from **1/8** with **Samsung Galaxy S10+(Android 9).
